### PR TITLE
Manage remote

### LINKED ISSUE #347: Manage remotes
It would be nice to be able to add, remove and manage remotes from the branch view.

### PR DESCRIPTION
**Pull request by stoivo**
Finish up and closes #347 

Now we have `git: remote add` and `git: remote remove`
There exist rename, but it will have to wait.

### CODE REVIEW

File: core/commands/remote.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    def on_select_remote(self, input_url):
+        self.url = input_url
+        # URLs can come in one of following formats format
+        # https://github.com/divmain/GitSavvy.git
+        #     git@github.com:divmain/GitSavvy.git
+        # Kind of funky, but does the job
+        _split_url = re.split('/|:', input_url)
+        owner = _split_url[-2] if len(_split_url) >= 2 else ''
+
+        self.view.window().show_input_panel("Remote name", owner, self.on_select_name, None, None)
```
Comment (divmain) [inline core/commands/remote.py]: I like how you pre-populated this!

File: core/commands/remote.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                flags=sublime.MONOSPACE_FONT,
+                selected_index=0
+                )
+
+    def on_selection(self, remotes_index):
+        if remotes_index == -1:
+            return
+
+        @util.actions.destructive(description="remove a remote")
+        def do_remote():
```
Comment (divmain) [inline core/commands/remote.py]: Rename this to `remove`.

**Comment by divmain** [comment]
Only needs one (very minor) change.  Thanks for this!

**Comment by stoivo** [comment]
@divmain, fixed

**Comment by divmain** [comment]
Fantastic.  Thanks @stoivo!

**Pull request merged**